dev-haskell/universe-base/universe-base-1.1.1.ebuild

```diff
@@ -26,12 +26,13 @@ RDEPEND=">=dev-haskell/nats-1.1.2:=[profile?] <dev-haskell/nats-1.2:=[profile?]
 "
 DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-1.18.1.3
-	test? ( >=dev-haskell/quickcheck-2.8.2 <dev-haskell/quickcheck-2.14 )
+	test? ( >=dev-haskell/quickcheck-2.8.2 <dev-haskell/quickcheck-2.15 )
```

Comment:
Upper bound should be dropped.

Reply:
I don't have a problem going in and removing the upper bounds.

Are there any concerns that this could break things in the future though? Or is that less of an issue, and worth dealing with on a case-by-case basis in order to ease the maintenance burden?

Reply:
> I don't have a problem going in and removing the upper bounds.
> 
> Are there any concerns that this could break things in the future though? Or is that less of an issue, and worth dealing with on a case-by-case basis in order to ease the maintenance burden?

Ideally hackage package should just work and it's depends should reflect supported ranges. But once we resort to patching I'd say it's fine to do patching in a way that would take less maintenance burden later as package is probably not well maintained.

> Furthermore, if that's the case, why does hackport include upperbounds at all?

It's a good question. The obvious answer is that `.cabal` file also contains upper bounds and thus is guaranteed to fail when we attempt to use unsatisfied upper bound of a depend. It's not very trivial to patch `.cabal` files automatically without breaking other things.

Reply:
I think this mostly makes sense 😀

Thanks for taking the time to review my questions.